sdk/template/template/src/configurationClient.ts

```diff
@@ -1,20 +1,19 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
 
+import { ConfigurationSetting, GeneratedClient } from "./generated";
 import {
+  InternalPipelineOptions,
+  OperationOptions,
   PipelineOptions,
   TokenCredential,
-  OperationOptions,
   bearerTokenAuthenticationPolicy,
-  createPipelineFromOptions,
-  InternalPipelineOptions
+  createPipelineFromOptions
 } from "@azure/core-http";
```

Comment:
Not related to this PR,  I didn't realize we are still using core-http in the template. I think we should update to corev2. Filed #19447 to track